jbpm-wb-showcase/pom.xml

```diff
@@ -1661,7 +1662,6 @@
                 <include>src/main/webapp/WEB-INF/lib/</include>
                 <include>src/main/webapp/WEB-INF/tlds/</include>
                 <include>.errai/</include>
-                <include>.niogit/**</include>
```

Comment:
@pefernan any particular reason to keep the `.niogit` folder? I found quite problematic to keep reusing it for development.